tests/lowering/eltwise/unary/test_hardsigmoid.py

```diff
@@ -0,0 +1,58 @@
+import torch
+import torch_ttnn
+import pytest
+import ttnn
+
+from tests.utils import assert_with_pcc
+
+
+class HardSigmoidModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x):
+        return torch.nn.functional.hardsigmoid(x)
+
+
+@pytest.mark.parametrize(
+    "input_shape",
+    (
+        (1, 72, 1, 1),
+        (1, 120, 1, 1),
+        (1, 480, 1, 1),
+        (1, 672, 1, 1),
+        (1, 256, 1, 1),
+        (1, 512, 1, 1),
+        (1, 768, 1, 1),
+        (1, 1024, 1, 1),
+        (1, 960, 1, 1),
+        (1, 16, 1, 1),
+        (1, 96, 1, 1),
+        (1, 240, 1, 1),
+        (1, 144, 1, 1),
+        (1, 288, 1, 1),
+        (1, 576, 1, 1),
+        (2, 3, 4, 5),
+        (7, 7, 7),
+        (420, 69),
+        pytest.param((1337,), marks=pytest.mark.xfail(reason="1D cases solved in #198, waiting for review")),
```

Comment:
I added 4 custom cases for robustness.  1-D elementwise unary ops occasionally fail.  I solved this in #198.

Reply:
Please fire a ticket so that I can report back to the team

Reply:
- tenstorrent/tt-metal#12671 ?

Reply:
Oh no, the workaround in #198 stops working.  I'll find another way when I'm working on `aten.ceil`.

Reply:
#198 is good again